src/main/resources/reference.conf

```diff
@@ -210,6 +210,14 @@ target {
       format = "json"
       path = ${common.input}"/target-inputs/ensembl/homo_sapiens_104.jsonl"
     }
+    gen-code = {
+      format = "csv"
+      path = ${common.input}"/target-inputs/gencode/gencode.v40.annotation.gff3.gz"
```

Comment:
We need to normalize this file name, if possible, by getting rid of the embedded version number 'v40'.

Reply:
Would it be possible to do it in this round? If not, that's fine, we could do it in future iterations.

Reply:
This is normalized at opentargets/platform-input-support#73 as _gencode.gff3.gz_
We just need to change _gencode.v40.._ to match PIS output.